src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/VariantOfListFW.java

```diff
@@ -0,0 +1,302 @@
+        Flyweight enumWithInt8 = enumWithInt8RO.tryWrap(buffer, offset, maxLimit);
+        if (enumWithInt8 == null)
+        {
+            return null;
+        }
+        switch (kind())
+        {
+        case ONE:
+            if (list32RO.tryWrap(buffer, offset + enumWithInt8.sizeof(), maxLimit) == null)
+            {
+                return null;
+            }
+            break;
+        case TWO:
+            if (list8RO.tryWrap(buffer, offset + enumWithInt8.sizeof(), maxLimit) == null)
+            {
+                return null;
+            }
+            break;
+        case THREE:
+            if (list0RO.tryWrap(buffer, offset + enumWithInt8.sizeof(), maxLimit) == null)
+            {
+                return null;
+            }
+            break;
+        default:
+            break;
+        }
+        if (limit() > maxLimit)
+        {
+            return null;
+        }
+        return this;
+    }
+
+    @Override
+    public int limit()
+    {
+        return get().limit();
+    }
+
+    public static final class Builder extends Flyweight.Builder<VariantOfListFW>
+    {
+        private final List32FW.Builder list32RW = new List32FW.Builder();
+
+        private final List8FW.Builder list8RW = new List8FW.Builder();
+
+        private final List0FW.Builder list0FW = new List0FW.Builder();
+
+        private final EnumWithInt8FW.Builder enumWithInt8RW = new EnumWithInt8FW.Builder();
+
+        public Builder()
+        {
+            super(new VariantOfListFW());
+        }
+
+        private Builder kind(
+            EnumWithInt8 value)
+        {
+            enumWithInt8RW.wrap(buffer(), offset(), maxLimit());
+            enumWithInt8RW.set(value);
+            limit(enumWithInt8RW.build().limit());
+            return this;
+        }
+
+        public Builder field(
+            Flyweight.Builder.Visitor mutator)
+        {
+            list32RW.field(mutator);
+            limit(list32RW.limit());
+            return this;
+        }
+
+        public Builder setAsList32(
+            ListFW list)
+        {
+            kind(KIND_ONE);
+            List32FW.Builder list32 = list32RW.wrap(buffer(), limit(), maxLimit());
+            list32.fields(list.fieldCount(), list.fields(), 0, list.length() - BitUtil.SIZE_OF_INT);
+            limit(list32.build().limit());
+            return this;
+        }
+
+        public Builder setAsList8(
+            ListFW list)
+        {
+            kind(KIND_TWO);
+            List8FW.Builder list8 = list8RW.wrap(buffer(), limit(), maxLimit());
+            list8.fields(list.fieldCount(), list.fields(), 0, list.length() - BitUtil.SIZE_OF_BYTE);
+            limit(list8.build().limit());
+            return this;
+        }
+
+        public Builder setAsList0(
+            ListFW list)
+        {
+            kind(KIND_THREE);
+            List0FW.Builder list0 = list0FW.wrap(buffer(), limit(), maxLimit());
+            list0.fields(list.fieldCount(), list.fields(), 0, 0);
```

Comment:
This line should be the same as the other cases, leaving it up to `list0` to validate, rather than passing literal `0` for last parameter, right?

If so, then suggest:
```
final DirectBuffer fields = list.fields();
list0.fields(list.fieldCount(), fields, 0, fields.capacity());
```
...and same feedback for `setAsList8` and `setAsList32` methods.